css/style.css

```diff
@@ -1,15 +1,455 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	overflow-x: hidden;
+}
+
 body {
 	font-family: 'GothamPro Bold';
+	margin: 0;
+	padding: 0;
 }
 
 a {
 	text-decoration: none;
+	display: inline-block;
+}
+
+/*header start*/
+.header-content {
+	max-width: 1170px;
+	margin: 0 auto;
 }
 
-.header__container {
+.header-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
 	max-width: 1030px;
+	height: 115px;
+	margin: 0 auto;
+}
+
+.header-container__inner {
+	flex: 1;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.header-logo {
+	width: 55px;
+	height: 55px;
+	margin-right: 46px;
+	margin-top: 2px;
+}
+
+.header-mobile__logo {
+	display: none;
+}
+
+.header-menu__humb {
+	display: none;
+}
+
+.header-logo__logo {
+	width: 100%;
+}
+
+.header-menu__list li {
+	list-style-type: none;
+	display: inline-block;
+	margin: 0;
+	padding: 0;
+}
+
+.header-menu__list{
+	white-space: nowrap;
+	margin: 0;
+	padding: 0;
+}
+
+.header-menu__link {
+	color: #000;
+	display: inline-block;
+	margin: 0;
+	padding: 1px 30px 0 0;
+	font-size: 12px;
+}
+
+.header-menu__list li:last-child {
+	padding-right: 0;
+}
+
+.header-container__wrap {
+	flex: 1;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	white-space: nowrap;
+	padding-top: 2px;
+}
+
+.header-search {
+	margin-right: 34px;
+}
+
+.header-search .header-search__form {
+	border-bottom: 1px solid #000;
+}
+
+.header-search__form {
+	position: relative;
+	max-width: 178px;
+}
+
+.header-search__search , 
+.header-search__btn {
+	border: none;
+	outline: none;
+	background: transparent;
+}
+
+.header-search__btn {
+	position: absolute;
+	height: 24px;
+	width: 24px;
+	top: 2px;
+	right: 0;
+	cursor: pointer;
+}
+
+.header-search__btn:before {
+	content: '\e900';
+	font-family: 'icomoon';
+	font-size: 15px;
 }
 
-.main__slider_container {
+.header-search__search {
+	width: 100%;
+	height: 24px;
+	opacity: 70%;
+}
+
+.header-links a {
+	color: #000;
+	font-size: 12px;
+}
+
+.header-links {
+	display: flex;
+	align-items: center;
+}
+
+.header-links__login {
+	margin-right: 35px;
+}
+
+.header-links__cart {
+	margin-right: 35px;
+}
+
+.header-links__cart-number {
+	display: inline-block;
+}
+
+.header-links__cart-number__text {
+	font-family: 'GothamPro Bold';
+	color: #fff;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-size: 10px;
+	width: 14px;
+	height: 14px;
+	border-radius: 50%;
+	border: 1px solid #000;
+	background: #000;
+}
+
+.header-links__location {
+	width: 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457136)